modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -34,6 +34,11 @@ a different storage backend, such as `NFS`, to configure the registry storage.
 . To configure your registry to use storage, change the `spec.storage.pvc` in the
 `configs.imageregistry/cluster` resource.
 +
+[NOTE]
+====
+When using shared storage such as NFS, it is strongly recommended to create Security Context Constraints (SCCs) that use `supplementalGroups` IDs, rather than `fsGroup` IDs. Refer to the NFS *Group IDs* documentation for details.
```

Comment:
SCC has specific meaning in openshift, it's the api `securitycontextconstraints.security.openshift.io`, so here I think we should use `Security Context.`

`fsGroup` only can define one group ID.

Reply:
How about add a link to `Group IDs`?

Reply:
> SCC has specific meaning in openshift, it's the api `securitycontextconstraints.security.openshift.io`, so here I think we should use `Security Context.`
> 
> `fsGroup` only can define one group ID.

Hi, @qinpingli. I am glad you pointed that out. I changed the text a bit to be more in line with how we describe the strategy in the annotation note (8) used in the first example in this topic: https://docs.openshift.com/container-platform/4.5/authentication/managing-security-context-constraints.html#security-context-constraints-example_configuring-internal-oauth

Reply:
> How about add a link to `Group IDs`?

Would be nice, yes, but unfortunately we are unable to include xrefs at the "module" level as it can break docs builds. As a workaround, I included the link at the end (which is at the "assembly" level).